code/ship/ship.cpp

```diff
@@ -757,6 +757,11 @@ static int lightningtype_match(char *p)
 #define SHIP_MULTITEXT_LENGTH 4096
 #define DEFAULT_DELTA_BANK_CONST	0.5f
 
+#define ASK_HELP_SHIELD_PERCENT 0.1 // percent shields at which ship will ask for help
+#define ASK_HELP_HULL_PERCENT 0.3   // percent hull at which ship will ask for help
+#define AWACS_HELP_HULL_HI 0.75     // percent hull at which ship will ask for help
+#define AWACS_HELP_HULL_LOW 0.25    // percent hull at which ship will ask for help
```

Comment:
The names of these macros could be improved to make it clear that these are the default values. Prefixing them with `DEFAULT_` could work.

Also, since you are already touching this, could you replace this with `const` values to make the code a little more modern.

Reply:
Will do!